tests/common/testbed.py

```diff
@@ -188,7 +188,7 @@ def write_line_break(self, data=None):
                       explicit_start=True, Dumper=IncIndentDumper)
 
     def get_testbed_type(self, topo_name):
-        pattern = re.compile(r'^(t0|t1|ptf|fullmesh|dualtor)')
+        pattern = re.compile(r'^(t0|t1|ptf|fullmesh|dualtor|t2-vs)')
```

Comment:
checking for "t2" should be enough instead of "t2-vs" right ?
it might allow other t2 variants to be supported as well